src/main/java/domain/board/Axis.java

```diff
@@ -0,0 +1,74 @@
+package domain.board;
+
+import static util.NumberState.NEGATIVE;
+import static util.NumberState.POSITIVE;
+import static util.NumberState.ZERO;
+import static util.NumberState.findNumberState;
+
+import util.NumberState;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum Axis {
+
+    POSITIVE_X(POSITIVE, ZERO) {
+        @Override
+        public List<BoardLocation> createAllPath(BoardLocation current, BoardVector boardVector) {
+            List<BoardLocation> path = new ArrayList<>();
+            for (int i = 1; i < boardVector.dx(); i++) {
+                path.add(current.moveX(i));
+            }
+            return path;
+        }
+    },
+    POSITIVE_Y(ZERO, POSITIVE) {
+        @Override
+        public List<BoardLocation> createAllPath(BoardLocation current, BoardVector boardVector) {
+            List<BoardLocation> path = new ArrayList<>();
+            for (int i = 1; i < boardVector.dy(); i++) {
+                path.add(current.moveY(i));
+            }
+            return path;
+        }
+    },
+    NEGATIVE_X(NEGATIVE, ZERO) {
+        @Override
+        public List<BoardLocation> createAllPath(BoardLocation current, BoardVector boardVector) {
+            List<BoardLocation> path = new ArrayList<>();
+            for (int i = -1; i > boardVector.dx(); i--) {
+                path.add(current.moveX(i));
+            }
+            return path;
+        }
+    },
+    NEGATIVE_Y(ZERO, NEGATIVE) {
+        @Override
+        public List<BoardLocation> createAllPath(BoardLocation current, BoardVector boardVector) {
+            List<BoardLocation> path = new ArrayList<>();
+            for (int i = -1; i > boardVector.dy(); i--) {
+                path.add(current.moveY(i));
+            }
+            return path;
+        }
+    };
```

Comment:
추상메서드 대신에 함수형 인터페이스를 활용해서 람다식을 이용해보면 어떨까요? 😃 

```suggestion
    POSITIVE_X(POSITIVE, ZERO, (current, boardVector) -> {
        List<BoardLocation> path = new ArrayList<>();
        for (int i = 1; i < boardVector.dx(); i++) {
            path.add(current.moveX(i));
        }
        return path;
    }),
    POSITIVE_Y(ZERO, POSITIVE, (current, boardVector) -> {
        List<BoardLocation> path = new ArrayList<>();
        for (int i = 1; i < boardVector.dy(); i++) {
            path.add(current.moveY(i));
        }
        return path;
    }),
    NEGATIVE_X(NEGATIVE, ZERO, (current, boardVector) -> {
        List<BoardLocation> path = new ArrayList<>();
        for (int i = -1; i > boardVector.dx(); i--) {
            path.add(current.moveX(i));
        }
        return path;
    }),
    NEGATIVE_Y(ZERO, NEGATIVE, (current, boardVector) -> {
        List<BoardLocation> path = new ArrayList<>();
        for (int i = -1; i > boardVector.dy(); i--) {
            path.add(current.moveY(i));
        }
        return path;
    }),
    ;
```